### PR TITLE
dev, 3.0, 3.1: add GC io limit

### PR DESCRIPTION
**Pull request by youjiali1995**
Signed-off-by: youjiali1995 <zlwgx1023@gmail.com>

### What is changed, added or deleted? 

Add GC io limit related document.

### What is the related PR or file link(s)? 

https://github.com/tikv/tikv/pull/5263

### Which version does your change affect? 

v3.0 and dev.

### CODE REVIEW

**Comment by MyonKeminta** [comment]
I suggest not merging this PR too early, since if time is sufficient, we may make it to merge dynamically-configuring of GC limit to 3.0.6.

**Comment by youjiali1995** [comment]
> I suggest not merging this PR too early, since if time is sufficient, we may make it to merge dynamically-configuring of GC limit to 3.0.6.

Yes.

File: dev/reference/garbage-collection/configuration.md [hunk 1]
```diff
@@ -113,3 +113,10 @@ update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_tim
 - 并行：使用 `tikv_gc_concurrency` 选项所指定的线程数，并行地向每个 Region 发送请求。
 - 自动并行：使用 TiKV 节点的个数作为线程数，并行地向每个 Region 发送请求。
 - 分布式：无需 TiDB 通过对 TiKV 发送请求的方式来驱动，而是每台 TiKV 自行工作。
+
+## 流控
+TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响, `0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改:
+
+```bash
+$ tikv-ctl --host=ip:port modify-tikv-config -m server -n gc.max_write_bytes_per_sec -v 10MB
```
Comment (MyonKeminta) [inline dev/reference/garbage-collection/configuration.md]: Please remove the `'$'` to keep consistent with our other documents
```suggestion
tikv-ctl --host=ip:port modify-tikv-config -m server -n gc.max_write_bytes_per_sec -v 10MB
```

File: v3.0/reference/garbage-collection/configuration.md [hunk 2]
```diff
@@ -113,3 +113,10 @@ update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_tim
 - 并行：使用 `tikv_gc_concurrency` 选项所指定的线程数，并行地向每个 Region 发送请求。
 - 自动并行：使用 TiKV 节点的个数作为线程数，并行地向每个 Region 发送请求。
 - 分布式：无需 TiDB 通过对 TiKV 发送请求的方式来驱动，而是每台 TiKV 自行工作。
+
+## 流控
+TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响, `0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改:
+
+```bash
+$ tikv-ctl --host=ip:port modify-tikv-config -m server -n gc.max_write_bytes_per_sec -v 10MB
```
Comment (MyonKeminta) [inline v3.0/reference/garbage-collection/configuration.md]:
```suggestion
tikv-ctl --host=ip:port modify-tikv-config -m server -n gc.max_write_bytes_per_sec -v 10MB
```
ditto

**Comment by TomShawn** [comment]
@youjiali1995 When the content is ready, please also update v3.1 as well, thanks~

**Comment by youjiali1995** [comment]
> @youjiali1995 When the content is ready, please also update v3.1 as well, thanks~

Ok.

**Comment by youjiali1995** [comment]
I will update this PR when https://github.com/tikv/tikv/pull/5769 is merged.

**Comment by youjiali1995** [comment]
@MyonKeminta @CaitinChen PTAL, thanks!

**Comment by MyonKeminta** [comment]
LGTM once https://github.com/tikv/tikv/pull/6036 is merged.

File: dev/reference/garbage-collection/configuration.md [hunk 3]
```diff
@@ -113,3 +113,13 @@ update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_tim
 - 并行：使用 `tikv_gc_concurrency` 选项所指定的线程数，并行地向每个 Region 发送请求。
 - 自动并行：使用 TiKV 节点的个数作为线程数，并行地向每个 Region 发送请求。
 - 分布式：无需 TiDB 通过对 TiKV 发送请求的方式来驱动，而是每台 TiKV 自行工作。
+
+## 流控
+
+TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响, `0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
```
Comment (CaitinChen) [inline dev/reference/garbage-collection/configuration.md]:
```suggestion
TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响，`0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
```

File: v3.0/reference/garbage-collection/configuration.md [hunk 4]
```diff
@@ -113,3 +113,13 @@ update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_tim
 - 并行：使用 `tikv_gc_concurrency` 选项所指定的线程数，并行地向每个 Region 发送请求。
 - 自动并行：使用 TiKV 节点的个数作为线程数，并行地向每个 Region 发送请求。
 - 分布式：无需 TiDB 通过对 TiKV 发送请求的方式来驱动，而是每台 TiKV 自行工作。
+
+## 流控
+
+TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响, `0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
```
Comment (CaitinChen) [inline v3.0/reference/garbage-collection/configuration.md]:
```suggestion
TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响，`0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
```

File: v3.1/reference/garbage-collection/configuration.md [hunk 5]
```diff
@@ -113,3 +113,13 @@ update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_tim
 - 并行：使用 `tikv_gc_concurrency` 选项所指定的线程数，并行地向每个 Region 发送请求。
 - 自动并行：使用 TiKV 节点的个数作为线程数，并行地向每个 Region 发送请求。
 - 分布式：无需 TiDB 通过对 TiKV 发送请求的方式来驱动，而是每台 TiKV 自行工作。
+
+## 流控
+
+TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响, `0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
```
Comment (CaitinChen) [inline v3.1/reference/garbage-collection/configuration.md]:
```suggestion
TiKV 在 3.0.6 版本开始支持 GC 流控，可通过配置 `gc.max-write-bytes-per-sec` 限制 GC worker 每秒数据写入量，降低对正常请求的影响，`0` 为关闭该功能。该配置可通过 tikv-ctl 动态修改：
```

**Pull request merged**